crates/rune-modules/src/base64.rs

```diff
@@ -0,0 +1,37 @@
+use base64::prelude::*;
+use rune::{ContextError, Module};
+
+/// Construct the `base64` module.
+pub fn module(_stdio: bool) -> Result<Module, ContextError> {
+    let mut module = Module::with_crate("base64")?;
+    module.item_mut().docs([
+        "Base64 encoding and decoding",
+        "",
+        "This module provides functions to encode & decode base64 data.",
+    ])?;
+
+    module.function_meta(decode)?;
+    module.function_meta(encode)?;
+    Ok(module)
+}
+
+/// Decode a base64 String into data
+///
+/// ```rune
+/// //assert_eq!(base64::decode("+uwgVQA=")?, b"\xFA\xEC\x20\x55\0");
```

Comment:
Is this test intentionally commented out? If it's intended to fail, maybe do something like this:

```
assert!(base64::decode("+uwgVQA=").is_err());
```

Reply:
I commented it out, because I initially implemented it for an older version of rune, where comparing bytes was not possible. It should work now?

Reply:
Not working now, I need to find the location to support to compare two `Vec<u8>` within rune...

Reply:
I will return rune Bytes which is most likely the more correct approach.